packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -312,8 +312,11 @@ export class LangChainEmbeddingWrapper implements Embedding {
   }
 }
 
-/** A default embedding useful for DocsQA. Note that this requires an OPENAI_API_KEY. */
-export const defaultEmbedding = new LangChainEmbeddingWrapper(new OpenAIEmbeddings());
+/** A default embedding useful for DocsQA. Note that this requires OPENAI_API_KEY to be set. */
+if (!process.env.OPENAI_API_KEY) throw new Error('OPENAI_API_KEY not set');
```

Comment:
Does this work? Since the const object is what's exported, I won't expect execution of this line to be related to importing the const value.

Reply:
I see what you mean. I think in practice it shouldn't matter as process.env should be immutable though. Or did you have another idea?

Reply:
I was thinking you could do something like what Matt wrote and then include this check inside the makeChunker function, but your point about process.env being immutable makes sense too.

Reply:
OK. I was going to add a `MarkdownSplitter` too in a followup so I can take another look when I do that.

Reply:
This will work, because the entire file is executed whenever anything is `import`ed from it.